Secret-agency-web/src/main/react/secret-agency/src/types/Agent.d.ts

```diff
@@ -9,4 +9,5 @@ export interface IAgent {
   // private DepartmentDTO department;
   readonly missionIds: number[];
   readonly reportIds: number[];
+  readonly rank: string;
```

Comment:
Na serveru je ten Enum napsaný tak, že vrací podle mě number.

Reply:
rank vracia string

Reply:
Tak to musí asi dělat ten serializátor do JSONu, nebo nevím, kdy se z toho stane String.